app/GraphQL/Mutations/UserMembershipMutation.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Models\Membership;
+use App\User;
+
+class UserMembershipMutation
+{
+    public function addVisitingMembership($rootValue, array $args): bool
+    {
+        $user = User::findOrFail($args['user_id']);
```

Comment:
This will throw an exception - how is it handled?

Reply:
It's not handled. As per any lighthouse exception, it will return the default internal error stating "No query results for model [App\User] 123". We could make a custom exception, but it seems a bit overkill, and in reality, if we do get this error, it is because someone is messing around with the front end or we aren't validating input correctly

Reply:
As a note, in UKTS at the moment we are using the same system of not catching these exceptions - https://github.com/VATSIM-UK/ukts-core-api/blob/master/app/Modules/Endorsement/Special/Mutations/RequestSpecialEndorsementHandler.php